src/index.ml

```diff
@@ -18,6 +18,19 @@ all copies or substantial portions of the Software. *)
 include Index_intf
 module Stats = Stats
 module Cache = Cache
+module Checks = Checks
+
+module Key = struct
+  module type S = Key
+
+  module String_fixed = Data.String_fixed
```

Comment:
I think `String_fixed` is only used in tests, maybe we should define it only in tests?

Reply:
We'd need to define it in two places in the tests (`test/unix` and `test/cli`), which is what pushed me into putting it here instead. I also think it makes sense to provide a default implementation here, but am happy to go back to the previous state if you disagree.

Reply:
Having a default implementation make sense, it's fine with me to keep them here.